mappings/net/minecraft/class_2577.mapping

```diff
@@ -1,5 +1,9 @@
 CLASS net/minecraft/class_2577
```

Comment:
maybe `ServerNetworkHandlerManager` considering that this stores a list of server network handlers

Reply:
I'll accept that name for now. Not sure if it's actually right.

Reply:
OK I started mapping related classes based off that assumption and it quickly got confusing. I'll push what I have so you can see.

Reply:
class_2893 is where I decided we probably need some better names for these.